File: Haskell/beginner/1017.hs
-- Importing printf
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables
    time_spent <- readLn :: IO Float
    avg_speed <- readLn :: IO Float

    -- Doing the calculations to find the necessary fuel
    let total_distance =  time_spent * avg_speed / 12

    -- Showing necessary fuel
    printf "%.3f\n" total_distance

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1017.c
/*Includes*/
#include <stdio.h>

/*Main Function*/
int main(){
    /*Variable declaration and initialization*/
    int time_spent, avg_speed;
    double necessary_fuel;

    /*Requesting values to user*/
    scanf("%d", &time_spent);
    scanf("%d", &avg_speed);

    /*Finding necessary fuel*/
    necessary_fuel = (time_spent * avg_speed) / 12.0;

    /*Printing necessary fuel*/
    printf("%.3lf\n", necessary_fuel);

    /*Ending*/
    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/